### DISCUSSION TITLE
$call.checkErrorLevel returned with 1

### DISCUSSION THREAD

**Discussion by tumibytes** [NREL/ReEDS-2.0#96] [Q&A: Execution]
Hi, ReEDS Team! I was able to run ReEDS just fine a few days ago with cases_test.csv as the input, but I tried to run the same test again and cannot get past a_writedata.gms. I did everything the same. I've attached the [gamslog.txt](https://github.com/NREL/ReEDS_OpenAccess/files/8110399/gamslog.txt) file here. Please let me know if there is any way to resolve this. Thank you!

This is where the model stops running [information pasted from gamslog.txt]:

--- . a_writedata.gms(45) 2 Mb
--- call python C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\\input_processing\writecapdat.py C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\ ExistingUnits_EIA-NEMS.gdx PrescriptiveBuilds_EIA-NEMS.gdx PrescriptiveRetire
***
*** $call.checkErrorLevel returned with 1
***
*** Error 343 in C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\runs\t1_p4_ercot_seq\a_writedata.gms
    Abort triggered by above statement
                                      
--- createmodel.gms(1) 2 Mb 1 Error
*** Status: Compilation error(s)
--- Job createmodel.gms Stop 02/21/22 10:26:50 elapsed 0:00:05.927

**Comment by vcelloho** [comment]
Can you check the content of the lstfiles folder? You'll want to view the last modified .lst file in taht folder and search for ****. This can sometimes provide more detail on the errors compared to the gamslog.txt file.

If you're running the test file with no modifications, you may want to reset any changes in your Git repo, on the off chance that you inadvertently made a change.

**Reply by tumibytes** [reply]
I will try resetting the repo later, then! 
This is what 1_inputs.lst shows (the only file in the lst folder):

  38  $call.checkErrorLevel 'python C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\\input_processing\writecapdat.py C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\ ExistingUnits_EIA-NEMS.gdx PrescriptiveBuilds_EIA-NEMS.gdx PrescriptiveRetirements_EIA-NEMS.gdx Nuke80RetireYear C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\runs\t1_p3_ercot_seq\inputs_case\ 0 -d 0'
****                      $343
**** LINE     45 INCLUDE     C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\runs\t1_p3_ercot_seq\a_writedata.gms
**** LINE      1 INPUT       C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\runs\t1_p3_ercot_seq\createmodel.gms
**** 343  Abort triggered by above statement

**** 1 ERROR(S)   0 WARNING(S)
GAMS 37.1.0  r07954d5 Released Nov 11, 2021 WEX-WEI x86 64bit/MS Windows - 02/21/22 10:10:03 Page 2

**Comment by vcelloho** [comment]
The error looks the same, with an error triggering on the python call. There's a couple of ways to directly debug a python error, one is to copy the command into a the command line or a batch file and run. Below is an example of what to run.

python C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\input_processing\writecapdat.py C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\ ExistingUnits_EIA-NEMS.gdx PrescriptiveBuilds_EIA-NEMS.gdx PrescriptiveRetirements_EIA-NEMS.gdx Nuke80RetireYear C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\runs\t1_p3_ercot_seq\inputs_case\ 0 -d 0

Alternatively you can open the appropriate python file and set the testing variables to the appropriate argument values and run directly. This can be helpful in stepping through a script to find an issue. In that case you'll want to open the following file.

C:\Users\etumi\Documents\GitHub\ReEDS_OA218\Default_ReEDS_Local\input_processing\writecapdat.py

**Reply by tumibytes** [reply]
I tried resetting the repo, and it looks like it's working again! Thank you again for the help :)
